pom.xml

```diff
@@ -33,11 +33,6 @@
                 <artifactId>slf4j-api</artifactId>
                 <version>2.0.16</version>
             </dependency>
-            <dependency>
-                <groupId>org.apache.logging.log4j</groupId>
-                <artifactId>log4j-slf4j2-impl</artifactId>
```

Comment:
By removing the implementation you are losing the ability to generate logs. Also fix the log settings location